cmd/crank/xpkg/push.go

```diff
@@ -42,10 +42,32 @@ const DefaultRegistry = "xpkg.upbound.io"
 
 // pushCmd pushes a package.
 type pushCmd struct {
+	// Arguments.
+	Image string `arg:"" help:"The OCI image name to push the package to."`
+
+	// Flags. Keep sorted alphabetically.
+	PackageFile string `short:"f" type:"existingfile" placeholder:"PATH" help:"The xpkg file to push. If not specified and only one xpkg file exists in current directory it will be used."`
+
+	// Internal state. These aren't part of the user-exposed CLI structure.
 	fs afero.Fs
+}
```

Comment:
This renders as:

```
$ go run cmd/crank/main.go xpkg push --help
Usage: crossplane xpkg push <image>

Push a package to a package registry.

Crossplane can be extended using packages. A Crossplane package is sometimes
called an xpkg. Crossplane supports configuration, provider and function
packages.

A package is an opinionated OCI image that contains everything needed to extend
Crossplane with new functionality. For example installing a provider package
extends Crossplane with support for new kinds of managed resource (MR).

This command pushes a package to a registry. Packages are pushed to
xpkg.upbound.io by default. You can override this by including a registry in
your OCI image name: for example index.docker.io/example/package:v1.0.0.

See https://docs.crossplane.io/latest/concepts/packages for more information.

Arguments:
  <image>    The OCI image name to push the package to.

Flags:
  -h, --help                 Show context-sensitive help.
      --verbose              Print verbose logging statements.
  -v, --version              Print version and quit.

  -f, --package-file=PATH    The xpkg file to push. If not specified and only
                             one xpkg file exists in current directory it will
                             be used.
```